### PR TITLE
More small changes

### PR DESCRIPTION
**Pull request by mdavidlawson**
Mostly around removing food related content since we might be going in a
different direction this year.

Fixes Issue #30

### CODE REVIEW

**Comment by mdavidlawson** [comment]
**Matt I didn't update the constant for the price this year. I'm not sure where that is, I'll check with Mayo tonight and get back to you on what that price will be.**

**Comment by kheaactua** [comment]
I'm unsure about the line "you can bring your own food", double checking with others.

File: htdocs/sites/mayofest15_dev/app/views/login/login.html [hunk 1]
```diff
@@ -53,7 +53,6 @@
 						<label for="name">Full Name<span class="req glyphicon glyphicon-asterisk"></span></label>
 						<input type="text" class="form-control" id="name" ng-model="user.data.name" name="name" placeholder="Your Name" ng-minlength="5" required>
 						<p ng-show="signUpForm.name.$invalid && !signUpForm.name.$pristine" class="help-block">We name your name.</p>
-						<!--<p ng-show="signUpForm.name.$error.minlength" class="help-block">Your name has got to be longer than that.</p>-->
```
Comment (kheaactua) [inline htdocs/sites/mayofest15_dev/app/views/login/login.html]: Why comment this out?

File: htdocs/sites/mayofest15_dev/app/views/home/info.html [hunk 2]
```diff
@@ -31,14 +31,12 @@ <h1 id="anchor-membership">Wait... Membership?</h1>
 	<h1>Food &amp; Drink</h1>
 	<sub>Note! New changes coming for 2015!</sub>
 
-	<p>Food is not included in the annual membership fee. Food will be available during the weekend for purchase. We have a in house chef (Chef Frenchy) heading up all in house food and depending on availability <b>we may also have some vendors up at the site this year selling food throughout the weekend</b> (More information to come on this soon). Of course, you can always bring your own food, but you'll need to provide your own cutlery</p>
+	<p>Food is not included in the annual membership fee. Food will be available during the weekend for purchase. Of course, you can always bring your own food, but you'll need to provide your own cutlery.</p>
```
Comment (kheaactua) [inline htdocs/sites/mayofest15_dev/app/views/home/info.html]: Not sure if people can bring there own food.. Well, not sure if we promote that.

File: htdocs/sites/mayofest15_dev/app/views/orders/order.html [hunk 3]
```diff
@@ -45,13 +45,8 @@
 		<button class="btn btn-primary" ng-show="order.actions.open" ng-click="order.doOpen($parent.actionCallback)">Modify Order</button>
 		<button class="btn btn-primary" ng-show="order.actions.checkout && hideActionCheckout!=1" ng-click="order.doCheckout($parent.actionCallback)">Checkout</button>
 		<button class="btn btn-primary" ng-show="order.actions.payNow && hideActionPayNow!=1" ng-click="order.doConfirm($parent.actionCallback)">Pay Now</button>
-		<!--<button class="btn btn-primary" ng-show="order.actions.recalc">Re-Calc</button>-->
```
Comment (kheaactua) [inline htdocs/sites/mayofest15_dev/app/views/orders/order.html]: I totally understand the "remove commented code", but I'd prefer these to be left in, as they are to be implemented one day maybe..  And I do sometimes activate them to perform some actions.  Usually I'd totally agree, but in this case I I think they're best left in.

**Comment by mdavidlawson** [comment]
I don't recall us deciding to outlaw personal food

**Comment by kheaactua** [comment]
(not sure how to reply)  Not outlawing, just not encouraging.  I thought this was the policy since 2013 as we were tired of picking up people's containers and stuff.  I have no preference.. Just going to give time to people to weigh in.

I'll merge this in the morning.. Starting to make mistakes.. Tired because someone lent me their tractor and property to help my horse.. and then I drove his tractor into his garage door.. and then spent hours trying to hide the damage - mostly by making things worse.

**Pull request merged**